core/include/gnuradio-4.0/annotated.hpp

```diff
@@ -81,13 +81,13 @@ struct BlockingIO {
  * @tparam denominator Bottom number in the input-to-output sample ratio.
  * @tparam isConst Specifies if the resampling ratio is constant or can be modified during run-time.
  */
-template<std::size_t numerator = 1LU, std::size_t denominator = 1LU, bool isConst = false>
+template<std::uint64_t numerator = 1U, std::uint64_t denominator = 1U, bool isConst = false>
 struct ResamplingRatio {
     static_assert(numerator > 0, "Numerator in ResamplingRatio must be >= 0");
-    static constexpr std::size_t kNumerator   = numerator;
-    static constexpr std::size_t kDenominator = denominator;
-    static constexpr bool        kIsConst     = isConst;
-    static constexpr bool        kEnabled     = !isConst || (kNumerator != 1LU) || (kDenominator != 1LU);
+    static constexpr std::uint64_t kNumerator   = numerator;
+    static constexpr std::uint64_t kDenominator = denominator;
+    static constexpr bool          kIsConst     = isConst;
```

Comment:
In the `Block` class numerator and denominator are declared as `uint32_t`. Same of stride.

Reply:
done